examples/entities_from_manifests.rs

```diff
@@ -13,4 +13,340 @@
+                    text_layout_info: self.text_bundle.text_layout_info.clone(),
+                    text_flags: self.text_bundle.text_flags.clone(),
+                    // This field isn't `Clone`, but that's fine because it will be recomputed
+                    calculated_size: ContentSize::default(),
+                    focus_policy: self.text_bundle.focus_policy.clone(),
+                    text: self.text_bundle.text.clone(),
+                    style: self.text_bundle.style.clone(),
+                    transform: self.text_bundle.transform.clone(),
+                    global_transform: self.text_bundle.global_transform.clone(),
+                    visibility: self.text_bundle.visibility.clone(),
+                    inherited_visibility: self.text_bundle.inherited_visibility.clone(),
+                    view_visibility: self.text_bundle.view_visibility.clone(),
+                    z_index: self.text_bundle.z_index.clone(),
+                    background_color: self.text_bundle.background_color.clone(),
+                },
+            }
+        }
+    }
+
+    impl DialogBox {
+        fn from_raw(raw: &RawDialogBox) -> Self {
+            Self {
+                text_bundle: TextBundle::from_section(raw.text.clone(), TextStyle::default()),
+            }
+        }
+    }
+
+    #[derive(Asset, Serialize, Deserialize, TypePath)]
+    pub struct RawDialogBoxManifest {
+        dialog_boxes: Vec<RawDialogBox>,
+    }
+
+    #[derive(Resource)]
+    pub struct DialogBoxManifest {
+        dialog_boxes: HashMap<Id<DialogBox>, DialogBox>,
+    }
+
+    impl Manifest for DialogBoxManifest {
+        type Item = DialogBox;
+        type RawItem = RawDialogBox;
+        type RawManifest = RawDialogBoxManifest;
+        type ConversionError = std::convert::Infallible;
+
+        const FORMAT: ManifestFormat = ManifestFormat::Ron;
+
+        fn get(&self, id: Id<DialogBox>) -> Option<&Self::Item> {
+            self.dialog_boxes.get(&id)
+        }
+
+        fn from_raw_manifest(
+            raw_manifest: Self::RawManifest,
+            _world: &mut World,
+        ) -> Result<Self, Self::ConversionError> {
+            let mut dialog_boxes = HashMap::default();
+
+            for raw_dialog_box in raw_manifest.dialog_boxes.iter() {
+                dialog_boxes.insert(
+                    Id::from_name(&raw_dialog_box.name),
+                    DialogBox::from_raw(raw_dialog_box),
+                );
+            }
+
+            Ok(Self { dialog_boxes })
+        }
+    }
+
+    pub fn spawn_dialog_boxes(mut commands: Commands, dialog_box_manifest: Res<DialogBoxManifest>) {
+        for dialog_box in dialog_box_manifest.dialog_boxes.values() {
+            commands.spawn(dialog_box.clone());
+        }
+    }
+}
+
+/// This module demonstrates the item-as-partial-bundle pattern.
+///
+/// This pattern is useful when you have a lot of duplicated data between items in the manifest.
+mod items_as_partial_bundle {
```

Comment:
I think for the partial bundle, I would use something closer to the `Item` from the simple example. That way, the manifest item can be a component in a larger bundle. I don't feel too strongly though.